Overview.bs

```diff
@@ -102,14 +102,22 @@ There are two different methods which can be used to incrementally transfer font
 Patch Subset {#patch-subset}
 ----------------------------
 
-The first method,
-<a href="patch-incxfer">Patch Subset</a>, uses a server which can generate binary updates to an existing font file.
+In the the first method, <a href="patch-incxfer">Patch Subset</a> a server generates binary patches which a
+client applies to a subset of the font in order to extend the coverage of that subset. The server
+is stateless, it does not maintain any session data for clients between requests. Thus when a client
```

Comment:
Good, and important to note for the privacy folks who are concerned about state.